src/town_cmd.cpp

```diff
@@ -2879,15 +2883,26 @@ static bool TryBuildTownHouse(Town *t, TileIndex tile)
 		/* Special houses that there can be only one of. */
 		t->flags |= oneof;
 
-		BuildTownHouse(t, tile, hs, house, random_bits);
+		/* Should the house be protected? */
+		bool protect = hs->extra_flags.Test(HouseExtraFlag::BuildingIsProtected);
+
+		BuildTownHouse(t, tile, hs, house, random_bits, protect);
 
 		return true;
 	}
 
 	return false;
 }
 
-CommandCost CmdPlaceHouse(DoCommandFlag flags, TileIndex tile, HouseID house)
+/**
+ * Place a house manually.
```

Comment:
"manually" seems temporary. It is a DoCommand, so who knows how it will be used now or in the future :) So I would just state: `Place a house` :)